selfdrive/car/subaru/values.py

```diff
@@ -148,6 +148,21 @@ def init_make(self, CP: car.CarParams):
       [SUBARU_VERSION_REQUEST],
       [SUBARU_VERSION_RESPONSE],
     ),
```

Comment:
whitelist if only for camera?

Reply:
I suspect gen1 will have everything on bus0, not just the camera. Gen2 is where they moved the powertrain to the other bus

Reply:
This is for the tester present camera fix, isn't it?

Reply:
ah, yea I'll whitelist it